upup/pkg/fi/cloudup/awstasks/natgateway.go

```diff
@@ -42,16 +45,59 @@ func (e *NatGateway) CompareWithID() *string {
 }
 
 func (e *NatGateway) Find(c *fi.Context) (*NatGateway, error) {
+	ngw, err := e.findNatGateway(c)
+	if err != nil {
+		return nil, err
+	}
+	if ngw == nil {
+		return nil, nil
+	}
+
+	actual := &NatGateway{
+		ID: ngw.NatGatewayId,
+	}
+	actual.Subnet = e.Subnet
+	if len(ngw.NatGatewayAddresses) == 0 {
+		// Not sure if this ever happens
```

Comment:
Lets drop a `glog` line in here then.. just for grins to see if it ever comes up